tests/test_fetcher_small.py

```diff
@@ -67,7 +67,11 @@ def test_hydrafetcher_get_evaluation_eval_num(self):
                     
                     # /meta.json
                     meta_url = f'https://hydra.vtr.tools/build/{build_num}/download/5/meta.json'
-                    payload = {"build_num": build_num}
+                    payload = {
+                        "build_num": build_num,
+                        "date": "2020-07-17T22:12:40",
+                        "board": "arty"
+                    }
```

Comment:
is there also a test for behavior when date and/or board info is missing?

Reply:
I've added a test [here](https://github.com/SymbiFlow/FPGA-Tool-Performance-Visualization-Library/pull/24/files#diff-597bd53f44ed33888a5c481743653bbeR213-R225).